docs/modules/operation/pages/deep-dive/topology/enlinkd/shell/introduction.adoc

```diff
@@ -13,3 +13,136 @@ ssh -p 8101 admin@localhost
+
+[source, console]
+ssh -p 8101 admin@localhost
+help opennms:generate-topology
+
+
+== opennms:delete-topology
+
+Delete generated topology.
+The topology is identified by category 'GeneratedNode'
+
+== opennms:enlinkd-snmp-collect
+
+Collect enlinkd snmp data
+
+[source, console]
+SYNTAX
+opennms:enlinkd-snmp-collect [options] trackerClass host.
+
+Tracker Classes Supported:
+
+[source, console]
+CdpCacheTable           Dot1dBase               Dot1qTpFdbTable         IsisSysObjectGroup      MtxrLldpRemTable        OspfIfTable
+CdpGlobalGroup          Dot1dBasePortTable      IpNetToMediaTable       LldpLocalGroup          MtxrNeighborTable       OspfNbrTable
+CiscoVtp                Dot1dStpPortTable       IsisCircTable           LldpLocalTable          OspfAreaTable           TimeTetraLldpRemTable
+CiscoVtpVlanTable       Dot1dTpFdbTable         IsisISAdjTable          LldpRemTable            OspfGeneralGroup
+
+Some output example:
+
+[source, console]
+admin@opennms()> opennms:enlinkd-snmp-collect OspfGeneralGroup clab-eos-ospf-s2
+.1.3.6.1.2.1.14.1.1 (ospfRouterId)= 10.0.0.2
+.1.3.6.1.2.1.14.1.2 (ospfAdminStat)= 1 (enabled)
+.1.3.6.1.2.1.14.1.3 (ospfVersionNumber)= 2
+.1.3.6.1.2.1.14.1.4 (ospfAreaBdrRtrStatus)= 2 (FALSE)
+.1.3.6.1.2.1.14.1.5 (ospfAreaASBdrRtrStatus)= 2 (FALSE)
+
+[source, console]
+admin@opennms()> enlinkd-snmp-collect OspfNbrTable clab-eos-ospf-s1
+		.1.3.6.1.2.1.14.10.1.1.10.1.0.2.0 (ospfNbrIpAddr)= 10.1.0.2
+		.1.3.6.1.2.1.14.10.1.2.10.1.0.2.0 (ospfNbrAddressLessIndex)= 0
+		.1.3.6.1.2.1.14.10.1.3.10.1.0.2.0 (ospfNbrRtrId)= 10.0.0.2
+		.1.3.6.1.2.1.14.10.1.1.10.1.0.10.0 (ospfNbrIpAddr)= 10.1.0.10
+		.1.3.6.1.2.1.14.10.1.2.10.1.0.10.0 (ospfNbrAddressLessIndex)= 0
+		.1.3.6.1.2.1.14.10.1.3.10.1.0.10.0 (ospfNbrRtrId)= 10.0.0.3
+
+[source, console]
+admin@opennms()> enlinkd-snmp-collect LldpRemTable clab-eos-ospf-s1
+.1.0.8802.1.1.2.1.4.1.1.4.0.1.1 (lldpRemChassisIdSubtype)= 4 (macAddress)
+.1.0.8802.1.1.2.1.4.1.1.5.0.1.1 (lldpRemChassisId)= 001c73717d99
+.1.0.8802.1.1.2.1.4.1.1.6.0.1.1 (lldpRemPortIdSubtype)= 5 (interfaceName)
+.1.0.8802.1.1.2.1.4.1.1.7.0.1.1 (lldpRemPortId)= Ethernet1
+.1.0.8802.1.1.2.1.4.1.1.8.0.1.1 (lldpRemPortDesc)= s2 -> s1
+.1.0.8802.1.1.2.1.4.1.1.9.0.1.1 (lldpRemSysName)= s2
+.1.0.8802.1.1.2.1.4.1.1.4.0.2.1 (lldpRemChassisIdSubtype)= 4 (macAddress)
+.1.0.8802.1.1.2.1.4.1.1.5.0.2.1 (lldpRemChassisId)= 001c73422183
+.1.0.8802.1.1.2.1.4.1.1.6.0.2.1 (lldpRemPortIdSubtype)= 5 (interfaceName)
+.1.0.8802.1.1.2.1.4.1.1.7.0.2.1 (lldpRemPortId)= Ethernet2
+.1.0.8802.1.1.2.1.4.1.1.8.0.2.1 (lldpRemPortDesc)= s3 -> s1
+.1.0.8802.1.1.2.1.4.1.1.9.0.2.1 (lldpRemSysName)= s3
+
+[source, console]
+admin@opennms()> enlinkd-snmp-collect Bridgease clab-eos-ospf-s1
+className Bridgease, not found a suitable collectionTracker~                                                                                                                             admin@opennms()> enlinkd-snmp-collect Dot1dBase clab-eos-ospf-s1
```

Comment:
The second admin@opennms content (after the tilde) appears far to the right of the console text in the build version (see below). Should it be on its own line?
<img width="1245" alt="Screen Shot 2023-01-16 at 10 30 09 AM" src="https://user-images.githubusercontent.com/59453630/212714377-92c23920-af7a-47b6-825d-c6c289ad8569.png">

Reply:
yeah this was a typo. I'm fixing in the incoming commit